packages/devtools_app/lib/src/shared/preferences.dart

```diff
@@ -12,10 +12,13 @@ class PreferencesController {
   final ValueNotifier<bool> _darkModeTheme = ValueNotifier(true);
   final ValueNotifier<bool> _vmDeveloperMode = ValueNotifier(false);
   final ValueNotifier<bool> _denseMode = ValueNotifier(false);
+  final _inspectorPreferences = InspectorPreferencesController();
```

Comment:
nit: when you have a private var and a getter, you should always put the private var below the public getter. The order should go:

// getter (if present)
// variable
// setter (if present)

also, let's group the InspectorPreferencesController getter and variable below the rest of the global settings.